website/project/model.py

```diff
@@ -162,6 +162,24 @@ class Comment(GuidStoredObject):
     # }
     reports = fields.DictionaryField(validate=validate_comment_reports)
 
+    # For Django compatibility
+    @property
+    def pk(self):
+        return self._id
+
+    @property
+    def absolute_api_v2_url(self):
+        return absolute_reverse('comments:comment-detail', kwargs={'comment_id': self._id})
+
+    # used by django and DRF
+    def get_absolute_url(self):
+        return self.absolute_api_v2_url
+
+    def return_content(self, auth):
+        if auth and self.is_deleted and self.user._id != auth.user._id:
```

Comment:
Does this imply that if no auth is provided (no proof of user identity), it will simply return the text of the comment unmodified? Is that intentional?